packages/backend/routes/index.ts

```diff
@@ -67,16 +68,20 @@ router.post('/debug-svix', (req, res) => {
     }
 });
 
-router.post('/slack-alert', async (req, res) => {
+router.post('/discord-alert', async (req, res) => {
```

Comment:
@hrutik7 Why has this endpoint been changed?

Reply:
This has nothing to do with the Slack / Discord integration. This endpoint is used internally. We dont need a equivalent of this for Discord.